src/main/kotlin/org/rust/ide/actions/RsCargoActionGroup.kt

```diff
@@ -0,0 +1,32 @@
+package org.rust.ide.actions
+
+import com.intellij.openapi.actionSystem.ActionGroup
+import com.intellij.openapi.actionSystem.AnAction
+import com.intellij.openapi.actionSystem.AnActionEvent
+import org.rust.cargo.icons.CargoIcons
+import org.rust.cargo.project.settings.toolchain
+import org.rust.cargo.util.modulesWithCargoProject
+import org.rust.ide.utils.isNullOrEmpty
+
+class RsCargoActionGroup : ActionGroup() {
+    val actions : Array<AnAction>
+
+    init {
+        templatePresentation.text = "_Cargo"
+        templatePresentation.description = "Run cargo command"
+        templatePresentation.icon = CargoIcons.ICON
+        templatePresentation.isEnabledAndVisible = true
+        isPopup = true
+        actions = arrayOf(RsCargoAction("clean"), RsCargoAction("check"), RsCargoAction("update"))
```

Comment:
Hm, I think that `cargo check` and `cargo clean` would fail if executed in root virual package of the multi-package workspace. Let's add `--all` flag by default?